Source/compositorclient/RPI/Implementation.cpp

```diff
@@ -193,6 +281,11 @@ class Platform {
     Platform()
     {
         bcm_host_init();
+        if (std::getenv("WPE_BCMRPI_CURSOR")) {
```

Comment:
Can you please use Core::SystemInfo::GetEnvironment ?

Reply:
Sure, I will do